CMake/setup.cmake

```diff
@@ -117,17 +117,22 @@ if(APPLE)
   set(HAVE_VSNPRINTF 1)
   set(HAVE_SCANDIR 1)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated")
+  if (NOT DEFINED ${CMAKE_OSX_DEPLOYMENT_TARGET})
+    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.8)
+  endif()
   if(FLTK_BACKEND_X11)
     if(NOT(${CMAKE_SYSTEM_VERSION} VERSION_LESS 17.0.0)) # a.k.a. macOS version ≥ 10.13
       set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_LIBCPP_HAS_THREAD_API_PTHREAD")
     endif(NOT(${CMAKE_SYSTEM_VERSION} VERSION_LESS 17.0.0))
   else()
     set(FLTK_COCOA_FRAMEWORKS "-framework Cocoa")
+    message(STATUS "CMAKE_OSX_DEPLOYMENT_TARGET: ${CMAKE_OSX_DEPLOYMENT_TARGET}")
+    message(STATUS "CMAKE_OSX_SYSROOT: ${CMAKE_OSX_SYSROOT}")
```

Comment:
Just a hint: logging contents of CMake variables should be done with `fl_debug_var(NAME)`. This has three advantages:
1. less typing
2. less potential for errors (you don't have to type the same text twice)
3. "nice" alignment of variable names and values if more than 1 variable is being logged

Sorry, I missed that in my previous review, but there's not need to change it since this would be commented out anyway...